usermatch/src/backend.js

```diff
@@ -0,0 +1,147 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+
```

Comment:
Since the schema is in another file, we need to import it.
`const userSchema = require('./schema')`